samples/java-fibonacci-action/README.md

```diff
@@ -35,11 +35,11 @@ To start the application locally, the `exec-maven-plugin` is used. Use the follo
 mvn compile exec:exec
 ```
 
-For further details see [Running a service locally](https://developer.lightbend.com/docs/akka-serverless/developing/running-service-locally.html) in the documentation.
+For further details see [Running a service locally](https://docs.kalix.io/developing/running-service-locally.html) in the documentation.
 
 ## Exercise the service
 
-With both the proxy and your application running, any defined endpoints should be available at `http://localhost:9000`. In addition to the defined gRPC interface, each method has a corresponding HTTP endpoint. Unless configured otherwise (see [Transcoding HTTP](https://docs.akkaserverless.dev/java/writing-grpc-descriptors-protobuf.html#_transcoding_http)), this endpoint accepts POST requests at the path `/[package].[entity name]/[method]`. For example, using `curl`:
```

Comment:
I'm wondering if it was a conscious choice to point to the docs in `.dev` here. Were we expecting used to check from `main` branch and then us the docs from .dev? 

Anyway, we can switch back to .dev, but for the moment I'm just picked docs.kalix.io